test/hotspot/jtreg/compiler/c2/irTests/TestIRFma.java

```diff
@@ -87,97 +88,125 @@ public void assertResult(double a, double b, double c) {
     @Test
     @IR(counts = {IRNode.FMSUB, "> 0"},
         applyIfCPUFeature = {"asimd", "true"})
+    @IR(counts = {IRNode.FMA_F, "= 1"},
```

Comment:
When you read these tests, it is a bit hard to know how that IRNode.XX do the matching.
In this case FMSUB is actually not an IRNode but an instruction, which I didn't understand.